buffer.go

```diff
@@ -33,6 +33,37 @@ func SingleTagBuffer(key string, value interface{}) *Buffer {
 	return b
 }
 
+// BuildBuffer is used to build a *Buffer that contains an arbitrary number
+// of tags. Sample usage:
+//
+//	bld := BuildBuffer()
+//	bld.Add("a", 1)
+//	bld.Add("b", 2)
+//	buf := bld.Finish()
+func BuildBuffer() BufferBuilder {
+	bld := BufferBuilder{b: &Buffer{}}
+	bld.b.init(0, 0)
+	return bld
+}
+
+// BufferBuilder is returned by BuildBuffer.
+type BufferBuilder struct {
+	b *Buffer
+}
+
+// Add a log tag to the buffer. If the key was added already, the value is
+// replaced.
+func (bld BufferBuilder) Add(key string, value interface{}) {
```

Comment:
`s/interface{}/any/` throughout, if you're feeling hip.

Reply:
Done.